Framework/Reflectometry/src/ReflectometryReductionOneAuto3.cpp

```diff
@@ -277,6 +288,10 @@ void ReflectometryReductionOneAuto3::init() {
   // Polarization correction
   declareProperty(std::make_unique<PropertyWithValue<bool>>("PolarizationAnalysis", false, Direction::Input),
                   "Apply polarization corrections");
+  declareProperty(std::make_unique<WorkspaceProperty<MatrixWorkspace>>("PolarizationEfficiencies", "", Direction::Input,
+                                                                       PropertyMode::Optional),
+                  "A workspace to be used for polarization analysis that contains the efficiency factors as "
+                  "histograms: P1, P2, F1 and F2 in the Wildes method and Pp, Ap, Rho and Alpha for Fredrikze.");
```

Comment:
Looks like you've added a new parameter in the middle of the list here. This will break any scripts calling the algorithm using positional arguments. If you know the user base of the algorithm quite well this isn't necessarily a problem but worth highlighting this in the release note I think at least - or if it doesn't matter just move to the end of the list?

Reply:
Ah, thanks, best to move to the end of the list then I think. I'd put it in the middle just because in the algorithm dialog it feels more user friendly to have it near the PolarizationAnalysis checkbox, but no good if it breaks previous scripts - that's a good thought. I'll move it to the end.

Reply:
This PR adds new parameters to three algorithms, so I've moved them to the end of the list in all cases. We may want to think about it longer term because I can see how it might potentially cause confusion, but it will definitely be fine for now and better to avoid breaking existing scripts I think. Let me know if what I've changed doesn't look any good.